pkg/controller/jaeger/jaeger_controller_test.go

```diff
@@ -4,6 +4,9 @@ import (
 	"context"
 	"testing"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Import order is strange.

Reply:
@jpkrohling isn't make fmt supposed to fix this?

Reply:
It's complicated :-)